fuel-tx/src/transaction/types/utxo_id.rs

```diff
@@ -20,10 +24,42 @@ use rand::{
 #[derive(Debug, Default, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 #[cfg_attr(feature = "typescript", wasm_bindgen::prelude::wasm_bindgen)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(transparent))]
+#[derive(fuel_types::canonical::Deserialize, fuel_types::canonical::Serialize)]
+pub struct CompressibleTxId(TxId);
```

Comment:
I think we just need to implement `CompressibleBy` for `UtxoId` itself, because in the `fuel-core` we have table `UtxoId` -> `Coin` where `Coin` contains `TxPointer`. In this case we don't need `CompressibleTxId`

Reply:
Makes sense. Changed to use whole `UtxoId` in b3e0718b.